glean-core/preview/README.md

```diff
@@ -0,0 +1,47 @@
+# glean-preview
+
+The `Glean SDK` is a modern approach for a Telemetry library and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
+
+## `glean-preview`
+
+
+This library provides a Rust API on top of Glean, targetted to Rust consumers.
```

Comment:
```suggestion
This library provides a Rust API on top of Glean, targeted to Rust consumers.
```

Reply:
I don't know why I keep making that mistake.